site/book/03-packages/05-updating-a-package.md

```diff
@@ -2,6 +2,24 @@ An independent package records the exact commit where the local fork and the ups
 diverged. This enables kpt to fetch any update to the upstream package and merge it with local
 changes.
 
+## Commit your local changes
+
+Before you update the package, you want to commit your local changes.
+
+First, to see the changes you've made to the fork of the upstream package:
+
+```shell
+$ git diff
+```
+
+If you're happy with the changes, commit them:
+
+```shell
+$ git add .; git commit -m "My changes"
```

Comment:
this is probably a thing for a different PR but I found that the git command need to be executed in the folder and the kpt commands are assumed to be executing from a parent directory.  I thinking any mid level engineer is going to pick up on that very quickly so it's not a blocking thing, but might to address that somehow later.

Reply:
We should clarify what the current working directory is when invoking commands. I try to be consistent if sequentially executing all commands in the book, but we can't really rely on a `cd` from few pages back.

Open to ideas to follow up here:
1) Annotate each block using a comment, e.g.:

```shell
// From wordpress/
$ kpt
```

2) always `cd` into a dir (not idemotent)